consensus/ibft/poa.go

```diff
@@ -50,7 +50,7 @@ func (poa *PoAMechanism) IsAvailable(hookType HookType, height uint64) bool {
 	case VerifyHeadersHook, ProcessHeadersHook, CandidateVoteHook:
 		return poa.IsInRange(height)
 	case PreStateCommitHook:
-		return poa.ibft.customContractAddress != types.ZeroAddress &&
+		return types.StringToAddress(poa.ibft.customContractAddress) != types.ZeroAddress &&
```

Comment:
if poa.ibft.customContractAddress is already an Address you wouldn't have to convert it here